ykrt/src/compile/jitc_yk/jit_ir.rs

```diff
@@ -1298,10 +1298,10 @@ impl LookupGlobalInst {
 pub struct CallInst {
     /// The callee.
     target: FuncDeclIdx,
-    /// The first argument to the call, if present. Undefined if not present.
```

Comment:
Looks like we could even drop the `#[repr(packed)]` now too?

Reply:
Unfortunately doing so tips is into 8, not 7, bytes.